views/random-beer.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="beer-container">
+    {{> beerCard beer}}
+    <ul>
+        {{#each food_pairing}}
+        <li>{{this}}</li>
+        {{/each}}
+    </ul>
+    <p>{{brewer_tips}}</p>
+</div>
```

Comment:
You don't actually need a "random-beer" view, you can just reuse the "beer-details" one.